kusd/api.go

```diff
@@ -170,6 +110,18 @@ func (api *PrivateValidatorAPI) GetMinimumDeposit() uint64 {
 	return api.kusd.GetMinimumDeposit()
 }
 
+// GetDeposits returns the validator deposits
+func (api *PrivateValidatorAPI) GetDeposits() ([]*types.Deposit, error) {
+	return api.kusd.Validator().Deposits()
+}
+
+func (api *PrivateValidatorAPI) RedeemDeposits() (bool, error) {
```

Comment:
Probably this could just return `error`.

Reply:
:+1:

Reply:
They use this pattern all over the API - example: https://github.com/ethereum/go-ethereum/blob/master/eth/api.go#L214. I think that in the end the goal is provide some input to the user - It's probably not the best way to it though. What do you think about it? @acroca @jmatosp

Reply:
I prefer just to return an error, the bool is really just `err != nil`